vmicore/src/lib/vmi/LibvmiInterface.cpp

```diff
@@ -342,16 +342,28 @@ namespace VmiCore
     }
 
     std::unique_ptr<std::string> LibvmiInterface::extractUnicodeStringAtVA(const addr_t stringVA, const addr_t cr3)
+    {
+        auto extractedString = tryExtractUnicodeStringAtVA(stringVA, cr3);
+
+        if (!extractedString)
+        {
+            throw VmiException(fmt::format("{}: Unable to convert unicode string", __func__));
+        }
+        return std::move(extractedString.value());
```

Comment:
Why are you using `std::move` here?

Reply:
In the best case it has no effect, in the worst case it pessismizes optimization opportunities for the compiler.